deploy/kubernetes/helm/che/values.yaml

```diff
@@ -119,6 +119,11 @@ che:
 #    pluginBroker:
 #      waitTimeoutMin: "3"
 #    pluginRegistryUrl: "https://che-plugin-registry.openshift.io/v3"
+#    tolerations:
```

Comment:
What about using `.Values.cheWorkspacePodTolerationsJson` instead of `.Values.che.workspace.tolerations` ?
It will be more consistent and we could have:
```
{{- if .Values.cheWorkspacePodTolerationsJson }}
  CHE_WORKSPACE_POD_TOLERATIONS__JSON: {{ .Values.cheWorkspacePodTolerationsJson | toJson | quote }}
{{- end }}
```

Reply:
That's not a problem.

Any idea if what @sleshchenko described [here](https://github.com/eclipse/che/pull/18562#discussion_r545674195) will be on your roadmap soon?

Reply:
The reason I'm asking this is I just tried to be consistent with `.Values.che.workspace.podNodeSelector`.

Reply:
@tolusha 
Should it be .Values.cheWorkspacePodTolerations instead, since we do not use JSON in the helm values?
```yaml
cheWorkspacePodTolerations:
  - key: "a.node.taint"
    operator: "Equal"
    value: "aValue"
    effect: "NoExecute"
```

Reply:
@cccs-eric 
yes, make sense.